pycode/memilio-epidata/memilio/epidata/getVaccinationData.py

```diff
@@ -280,7 +304,7 @@ def get_vaccination_data(read_data=dd.defaultDict['read_data'],
     # get number of new age groups that are not vaccinated at all
     j = 0
     new_age_not_vacc = 0
-    while min_age_new[j+1] < min_age_old[0]:
+    while min_age_new[j+1] <= min_age_old[0]:
```

Comment:
This was causing the error in the old functionality. I hope that this now will work for the future if other age groups are added or if the current ones are changed. Could you also think about it @annette-lutz ?

Reply:
I think this should work. Maybe it is a good idea to impelement a unittest for data with different agegroups.

Reply:
Like in the most recent commit?